web/beacon-app/src/components/auth/LandingFooter/LandingFooter.tsx

```diff
@@ -9,10 +9,10 @@ import TwitterIcon from '@/assets/icons/twitterIcon';
 function LandingFooter() {
   const { version: appVersion, revision: gitRevision } = appConfig;
   return (
-    <footer className="bg-footer bg-cover bg-no-repeat text-white ">
-      <div className="pt-72 2xl:pt-80">
+    <footer className="mt-20 bg-footer bg-cover bg-no-repeat text-white sm:mt-0 ">
+      <div className="pt-[300px] sm:pt-64 lg:pt-[225px] 2xl:pt-[320px]">
```

Comment:
Without padding, the text below is positioned closer to the top of the footer element where the tower begins in the bg image. That area is full of whitespace, so the padding is pushing the `div` element containing the text down so that it appears above the blue background after the waves begin.

The `bg-cover` style was applied to make the bg image span the full width of a screen. Unfortunately, this causes the size of the waves to expand as the screen width increases. This then causes the text to sometimes appear above the waves, so that's why the padding changes depending on the screen size.

There may be a way to resolve the issue by cropping an image of the waves and then using it as a background to the bg image, but that will take much longer, so a follow on story will need to be created.

Reply:
Ok, I understand -- let's keep it the way it is, worrying about ultra wide resolution is a pretty low priority. The fixes you've made so far have a higher impact. I don't think we need to make a follow on story for a cropped image of the waves.